docs/dev/tutorials/aa-daily-spend-limit.md

```diff
@@ -61,36 +64,34 @@ module.exports = {
         },
     },
     solidity: {
-        version: "0.8.17",
+        version: "0.8.13",
     },
 };
-
 ```
 
 ::: tip zksync-cli
-
-You can use the zkSync CLI to scaffold a project automatically. Find [more info about the zkSync CLI here](../../api/tools/zksync-cli/).
-
+- Use the zkSync Era CLI to scaffold a project automatically. 
```

Comment:
```suggestion
- You can use the zkSync Era CLI to scaffold a project automatically. 
```
This could confuse users into thinking they have to use the CLI at this point (?)